README.md

```diff
@@ -1,2 +1,34 @@
 # hw01
-Repository for first CFSS homework assignment
+# Nora C. Nickels  
+
+### ![](https://0.academia-photos.com/30174041/13103076/14424279/s200_nora.nickels.png) 
```

Comment:
Not sure why this was placed on a line with a header. Should just be

```
![](https://0.academia-photos.com/30174041/13103076/14424279/s200_nora.nickels.png) 
```

Headers only work with text.